test/core/bulk.wast

```diff
@@ -193,6 +192,9 @@
   (data "")
   (func (data.drop 64)))
 
+;; No memory is required for the data.drop instruction.
+(module (data "") (func (data.drop 0)))
```

Comment:
Maybe use a non-empty segment, so that the drop isn't a nop.

Reply:
Done.